go.mod

```
@@ -0,0 +1,77 @@
+module github.com/paketo-buildpacks/builder-noble-buildpackless-tiny
+
+go 1.22
+
+require (
+	github.com/onsi/gomega v1.33.1
+	github.com/paketo-buildpacks/occam v0.18.5
```

There are 12 newer dependencies, can you update go.mod?
```suggestion
	github.com/onsi/gomega v1.34.2
	github.com/paketo-buildpacks/occam v0.18.8
```